src/PointwiseFunctions/Elasticity/ConstitutiveRelations/IsotropicHomogeneous.hpp

```diff
@@ -0,0 +1,104 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+#include <limits>
+
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "Options/Options.hpp"
+#include "Utilities/TMPL.hpp"
+
+/// \cond
+namespace PUP {
+class er;
+}  // namespace PUP
+/// \endcond
+
+namespace Elasticity {
+/*!
+ * \brief Constitutive relations for elastic materials
+ */
+namespace ConstitutiveRelations {
+
+/*!
+ * \brief An isotropic and homogeneous material
+ *
+ * \details For an isotropic and homogeneous material the linear constitutive
+ * relation \f$T^{ij}=\chi^{ijkl}S_{kl}\f$ reduces to
+ * \f[
+ * \chi^{ijkl} = \lambda \delta^{ij}\delta^{kl} + \mu
+ * \left(\delta^{ik}\delta^{jl} + \delta^{il}\delta^{jk}\right) \\
+ * \implies \quad T^{ij} = \lambda \mathrm{Tr}(S) \delta^{ij} + 2\mu S^{ij}
```

Comment:
Can you please cit an equation for each relation here? Just below Eq. (11.19) in Thorne and Blandford, they have the same relation as you for T^{ij} except the right hand side differs from this by a sign, as does the RHS of Thorne and Blandford Eq. (11.18). Are you sure about the sign here? I would have thought, as Thorne and Blandford argue, that you need the minus signs so the material resists deformation when stressed, analogously to the minus sign in Hooke's law.

Reply:
I'll have to check the book, but I think the minus sign you mean you can find in the documentation of the `Elasticity` namespace (`Elasticity/Tags.hpp`):
$-\nabla_i T^{ij} = F^j$
I considered it part of the e.o.m. as opposed to the constitutive relation.